zenodo/modules/records/templates/zenodo_records/box/files.html

```diff
@@ -62,6 +63,12 @@
       </tbody>
     </table>
   {%- endif %}
+    <hr/>
```

Comment:
We also need to display ``{{record.access_conditions}}`` when mode is restricted access. See e.g. https://zenodo.org/record/55554